roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -50,14 +49,12 @@
 
 # Inject ownerReferences
 - name: Inject owner references for KubevirtCommonTemplatesBundle
-  include_role: 
-    name: ClaimOwnership
-  vars:
-    object: "{{ item }}"
-    objectKind: "templates"
-    owner: "{{ cr_info }}"
-  when: "{{ old_cr_exists==true }}"
-  with_items: "{{ old_cr_templates }}" # Templates
+  k8s:
+    state: present
+    namespace: "{{ cr_info.metadata.namespace }}"
+    merge_type: ['merge', 'json']
+    resource_definition: "{{ old_cr_templates | k8s_inject_ownership(cr_info) }}"
+  when: "{{ old_cr_templates | length > 0 }}"
```

Comment:
I looked at travis logs and it fails because `old_cr_templates` is not defined, probably because `old_cr_exists = false`, I think `(old_cr_templates is defined) and (old_cr_templates | length > 0)` might solve this